input/pagecontent/3-2-2-heartbeat.md

```diff
@@ -1,4 +1,4 @@
-> Implementator feedback is requested for the need to support heartbeats for websockets.
+> Implementator feedback is requested for the need to support heartbeats for WebSockets.

```

Comment:
Typo: *Implementator* should be Implementor

Reply:
Corrected